code/src/components/CheckboxInput.js

```diff
@@ -0,0 +1,34 @@
+import React from "react"
+
+const CheckboxInput = ({ choice, setChoice, data }) => {
+  const handleChange = (event) => {
+    choice.includes(event.target.value)
+      ? setChoice(
+        choice.filter((item) => item !== event.target.value)
+      )
+    : setChoice([event.target.value, choice])
+  }
+  return (
+    <div className="check-input">
```

Comment:
I know checkboxes are hard to make so good that you made them. There is however an error: if you check something and get the result and then you press Again?-button to make another one, the checkboxes do not look checked but the previously selected value sticks and the new values you check are just added on to that.